config/initializers/paperclip.rb

```diff
@@ -0,0 +1,14 @@
+Paperclip::Attachment.default_options.update({
+  s3_host_name: "s3-eu-west-1.amazonaws.com",
+  s3_protocol: "", # relative
+  url: "/system/:tenant/:class/:attachment_name/:id_partition/:style/:filename",
+  path: ":rails_root/public/system/:tenant/:class/:attachment_name/:id_partition/:style/:filename"
+});
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.<br>Do not use semicolons to terminate expressions.